package.json

```diff
@@ -53,46 +53,24 @@
-		"makepot:android:layout-grid": "npm run wp -- i18n make-pot ./block-experiments/blocks/layout-grid --include=*.native.js,*.android.js --exclude=test/*,e2e-tests/*,build/*,build-module/*,build-style/* --skip-php --subtract=./layout-grid.pot --domain=layout-grid layout-grid-android.pot",
-		"makepot:ios:gutenberg-mobile": "npm run wp -- i18n make-pot ./src --include=*.js,*.native.js,*.ios.js --exclude=test/* --subtract=./gutenberg.pot --ignore-domain gutenberg-mobile-ios.pot",
-		"makepot:ios:gutenberg": "npm run wp -- i18n make-pot ./gutenberg/packages --include=*.native.js,*.ios.js --exclude=test/*,e2e-tests/*,build/*,build-module/*,build-style/* --skip-php --subtract=./gutenberg.pot --ignore-domain gutenberg-ios.pot",
-		"makepot:ios:jetpack": "npm run wp -- i18n make-pot ./jetpack/projects/plugins/jetpack/extensions --include=*.native.js,*.ios.js --exclude=test/*,e2e-tests/*,build/*,build-module/*,build-style/* --skip-php --subtract=./jetpack.pot --domain=jetpack jetpack-ios.pot",
-		"makepot:ios:layout-grid": "npm run wp -- i18n make-pot ./block-experiments/blocks/layout-grid --include=*.native.js,*.ios.js --exclude=test/*,e2e-tests/*,build/*,build-module/*,build-style/* --skip-php --subtract=./layout-grid.pot --domain=layout-grid layout-grid-ios.pot",
-		"premakepot:gutenberg": "npm run clean:gutenberg && npm run build:gutenberg",
-		"makepot:gutenberg": "npm run wp -- i18n make-pot ./gutenberg/packages --exclude=test/*,e2e-tests/*,build/*,build-module/*,build-style/*,*.native.js,*.ios.js,*.android.js,bundle/* --ignore-domain gutenberg.pot",
-		"postmakepot:gutenberg": "npm run clean:gutenberg",
-		"makepot:jetpack": "npm run wp -- i18n make-pot ./jetpack/projects/plugins/jetpack/extensions --exclude=test/*,e2e-tests/*,build/*,build-module/*,build-style/*,*.native.js,*.ios.js,*.android.js,bundle/* --domain=jetpack jetpack.pot",
-		"makepot:layout-grid": "npm run wp -- i18n make-pot ./block-experiments/blocks/layout-grid --exclude=test/*,e2e-tests/*,build/*,build-module/*,build-style/*,*.native.js,*.ios.js,*.android.js,bundle/* --domain=layout-grid layout-grid.pot",
-		"pregenstrings": "npm run makepot:gutenberg && npm run makepot:jetpack && npm run makepot:layout-grid",
-		"genstrings": "npm run genstrings:android && npm run genstrings:ios",
-		"genstrings:android": "npm run makepot:android:gutenberg && npm run makepot:android:gutenberg-mobile && npm run makepot:android:jetpack && npm run makepot:android:layout-grid && ./bin/po2android.js bundle/android/strings.xml gutenberg-android.pot gutenberg-mobile-android.pot jetpack-android.pot layout-grid-android.pot",
-		"genstrings:ios": "npm run makepot:ios:gutenberg && npm run makepot:ios:gutenberg-mobile && npm run makepot:ios:jetpack && npm run makepot:ios:layout-grid && ./bin/po2swift.js bundle/ios/GutenbergNativeTranslations.swift gutenberg-ios.pot gutenberg-mobile-ios.pot jetpack-ios.pot layout-grid-ios.pot",
```

Comment:
These commands are no longer required as this logic is covered in the `generate-pot-files` script ([reference](https://github.com/WordPress/gutenberg/blob/trunk/packages/react-native-editor/bin/generate-pot-files.sh)).